fw/Core/Hitcon/Logic/Display/display.cc

```diff
@@ -102,6 +107,16 @@ void display_get_frame_packed(display_buf_t *buf, int frame) {
       break;
   }
 
+  if (display_orientation) {
```

Comment:
This no longer works after Alex's change, you'll need to use display_buf_assign(), see examples above.